cumulus/parachains/runtimes/collectives/collectives-westend/src/fellowship/mod.rs

```diff
@@ -0,0 +1,239 @@
+	type InductOrigin = EitherOfDiverse<
+		EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+		EitherOfDiverse<
+			pallet_ranked_collective::EnsureMember<
+				Runtime,
+				FellowshipCollectiveInstance,
+				{ ranks::DAN_3 },
+			>,
+			Members,
+		>,
+	>;
+	// Approval (rank-retention) of a Member's current rank is by any of:
+	// - Root;
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	// - a vote by the rank two above the current rank for all retention up to the Master rank.
+	type ApproveOrigin = EitherOf<
+		MapSuccess<
+			EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+			Replace<ConstU16<{ ranks::DAN_9 }>>,
+		>,
+		EnsureCanRetainAt,
+	>;
+	// Promotion is by any of:
+	// - Root can promote arbitrarily.
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	// - a vote by the rank two above the new rank for all promotions up to the Master rank.
+	type PromoteOrigin = EitherOf<
+		MapSuccess<
+			EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+			Replace<ConstU16<{ ranks::DAN_9 }>>,
+		>,
+		EnsureCanPromoteTo,
+	>;
+	type EvidenceSize = ConstU32<65536>;
+}
+
+pub type FellowshipSalaryInstance = pallet_salary::Instance1;
+
+use xcm::prelude::*;
+
+parameter_types! {
+	// The interior location on AssetHub for the paying account. This is the Fellowship Salary
+	// pallet instance (which sits at index 64). This sovereign account will need funding.
+	pub Interior: InteriorMultiLocation = PalletInstance(64).into();
+}
+
+const USDT_UNITS: u128 = 1_000_000;
+
+/// [`PayOverXcm`] setup to pay the Fellowship salary on the AssetHub in USDT.
+pub type FellowshipSalaryPaymaster = PayOverXcm<
+	Interior,
+	crate::xcm_config::XcmRouter,
+	crate::PolkadotXcm,
+	ConstU32<{ 6 * HOURS }>,
+	AccountId,
+	(),
+	ConvertToValue<UsdtAssetHub>,
+	AliasesIntoAccountId32<(), AccountId>,
+>;
+
+impl pallet_salary::Config<FellowshipSalaryInstance> for Runtime {
+	type WeightInfo = weights::pallet_salary_fellowship_salary::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Paymaster = FellowshipSalaryPaymaster;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Paymaster = PayWithEnsure<FellowshipSalaryPaymaster, OpenHrmpChannel<ConstU32<1000>>>;
+	type Members = pallet_ranked_collective::Pallet<Runtime, FellowshipCollectiveInstance>;
+
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Salary = pallet_core_fellowship::Pallet<Runtime, FellowshipCoreInstance>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Salary = frame_support::traits::tokens::ConvertRank<
+		crate::impls::benchmarks::RankToSalary<Balances>,
+	>;
+	// 15 days to register for a salary payment.
+	type RegistrationPeriod = ConstU32<{ 15 * DAYS }>;
+	// 15 days to claim the salary payment.
+	type PayoutPeriod = ConstU32<{ 15 * DAYS }>;
+	// Total monthly salary budget.
+	type Budget = ConstU128<{ 100_000 * USDT_UNITS }>;
```

Comment:
This seems to just set some `Balance`, but where does it define the `AssetId`?

Reply:
The asset ID is defined [here](https://github.com/paritytech/polkadot-sdk/blob/donal-prepare-sp-native-runtime-removal/cumulus/parachains/runtimes/collectives/collectives-westend/src/fellowship/mod.rs#L212) which is paid over XCM through the `FellowshipSalaryPaymaster`.

I just realised though, for `collectives-westend` I'm guessing we should actually set that to some amount of WND instead of USDT.

Reply:
That's OK, I saw this and created a test asset with ID 1984 :). Just ping me if I should mint / send to some account for testing.
https://westmint.statescan.io/#/extrinsics/5623802-2

Reply:
great, thanks